sdk/tables/azure-data-tables/migration_guide.md

```diff
@@ -0,0 +1,309 @@
+# Guide for migrating to `azure-data-tables` from `azure-cosmosdb-table`
+
+This guide is intended to assist in the migration to `azure-data-tables` from `azure-cosmosdb-table`. It will focus on side-by-side comparisons for similar operations between the two packages.
+
```

Comment:
What about people using the old table storage SDK?

Reply:
I'm not aware of an old table storage SDK, which one are you referring to?

Reply:
Hmmm, good question. I know there's an existing table storage *service* and I was assuming that there was a corresponding SDK to go with it, but I haven't actually seen it. So if one doesn't actually exist then you can ignore my comment.